docs/source/reference/yaml-spec.rst

```diff
@@ -776,7 +776,7 @@ Example:
       source: /local/path/datasets  # Source path, can be local or bucket URI. Optional, do not specify to create an empty bucket.
       store: s3  # Could be either 's3', 'gcs', 'azure', 'r2', 'oci', or 'ibm'; default: None. Optional.
       persistent: True  # Defaults to True; can be set to false to delete bucket after cluster is downed. Optional.
-      mode: MOUNT  # Either MOUNT or COPY. Defaults to MOUNT. Optional.
+      mode: MOUNT  # How buckets are mounted to nodes. MOUNT or COPY or MOUNT_CACHED. Defaults to MOUNT. Optional.
```

Comment:
The word `mounted` is a bit confusing here. How about just leave the options here.
```suggestion
      mode: MOUNT  # MOUNT or COPY or MOUNT_CACHED. Defaults to MOUNT. Optional.
```